tests/commonlib/io_utils.py

```diff
@@ -129,17 +131,62 @@ def exec_command(container_name: str, command: str, param_value: str, resource:
         if command == 'chmod':
             os.chmod(path=resource, mode=int(param_value, base=8))
         elif command == 'chown':
-            uid_gid = param_value.split(':')
-            if len(uid_gid) != 2:
-                raise Exception(
-                    "User and group parameter shall be separated by ':' ")
-            shutil.chown(path=resource, user=uid_gid[0], group=uid_gid[1])
+            try:
+                uid, gid = param_value.split(':')
+            except ValueError as exc:
+                raise Exception("User and group parameter shall be separated by ':' ") from exc
+
+            FsClient.add_users_to_node([uid, gid], in_place=True)
+            shutil.chown(path=resource, user=uid, group=gid)
         elif command == 'unlink':
             if not Path(param_value).is_dir():
                 Path(param_value).unlink()
         else:
             raise Exception(
                 f"Command '{command}' still not implemented in test framework")
+    
+    @staticmethod
+    def add_users_to_node(users: list, in_place: bool):
+        """
+        This function creates the given users along with groups with the
+        same name, on the local container as well the host node.
+        @param users: List of users to create.
+        @param in_place: Whether host node configuration files should be modified in-place or overwritten.
+        @return: None
+        """
+        if in_place:
+            host_users_file = Path('/hostfs/etc/passwd')
+            host_groups_file = Path('/hostfs/etc/group')
+
+            temp_etc = Path('/tmp/etc')
+            temp_etc.mkdir(parents=True, exist_ok=True)
+
+            temp_users_file = temp_etc / 'passwd'
+            temp_groups_file = temp_etc / 'group'
+
+            shutil.copyfile(host_users_file, temp_users_file)
+            shutil.copyfile(host_groups_file, temp_groups_file)
```

Comment:
Seems like we perform same operations for host_users_file and host_group_file. We can decrease number of lines by executing for loop on list of paths

Reply:
A loop of two iterations doesn't seem much better/cleaner to me than just running those two iterations.